src/cli/cli-definitions.js

```diff
@@ -174,5 +174,15 @@ export default {
   "verbose": {
     env: "VERBOSE",
     help: "Set the logging to verbose"
+  },
+  "logLevel": {
+    help: "Sets the log level"
+  },
+  "enableConfigChanges": {
+    env: "ENABLE_CONFIG_CHANGES",
```

Comment:
Also, what about defaulting to true? Then the dashboard can just work out of the box without needing to change any config?